python/lsst/afw/fits/_fits.cc

```diff
@@ -276,8 +276,9 @@ PYBIND11_MODULE(_fits, mod) {
     wrappers.addSignatureDependency("lsst.pex.exceptions");
     wrappers.addSignatureDependency("lsst.daf.base");
     // FIXME: after afw.image pybind wrappers are converted
-    //wrappers.addSignatureDependency("lsst.afw.image"); 
-    wrappers.wrapException<FitsError, lsst::pex::exceptions::IoError>("FitsError", "IoError");
+    //wrappers.addSignatureDependency("lsst.afw.image");
+    auto cls = wrappers.wrapException<FitsError, lsst::pex::exceptions::IoError>("FitsError", "IoError");
+    cls.def(py::init<std::string const &>());
```

Comment:
What does this do?

Reply:
Adding a ctor for the bound class, otherwise this happens

```
>>> from lsst.afw.fits import FitsError
>>> raise FitsError("test")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/wittgen/lsst/lsst_stack/stack/miniconda3-py38_4.9.2-4.1.0/Linux64/pex_exceptions/g48ccf36440+89c08d0516/python/lsst/pex/exceptions/wrappers.py", line 70, in __init__
    cpp = self.WrappedClass(message, *args, **kwds)
TypeError: lsst.afw.fits._fits.FitsError: No constructor defined!

```